aria-practices.html

```diff
@@ -710,100 +710,102 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
       <p>
-        With any form of list autocomplete, the popup may appear and disappear as the user types.
+        If a combobox is editable and has any form of list autocomplete, the popup may appear and disappear as the user types.
         For example, if the user types a two character string that triggers five suggestions to be displayed but then types a third character that forms a string that does not have any matching suggestions,
         the popup may close and, if present, the inline completion string disappears.
       </p>
       <p>
-        When constructing a widget that is both visually compact and enables users to choose one value from a set of discrete values, often either a <a href="#Listbox">listbox</a> or <a href="#menubutton">menu button</a> is simpler to implement and use.
-        One feature of combobox that distinguishes it from both listbox and menu button is that the value of the combobox is presented in an edit field.
-        Thus, the combobox gives users one function that both listbox and menu button lack, namely the ability to select some or all of the value for copying to the clipboard.
-        One feature that distinguishes both combobox and menu button widgets from listbox widgets is their ability to provide an undo mechanism.
-        In many implementations, users can navigate the set of allowed values in a combobox or menu and then decide to revert to the value the widget had before navigating by pressing <kbd>escape</kbd>.
-        In contrast, navigating a listbox immediately changes its value, and escape does not provide an undo mechanism.
-      </p>
-      <p class="note">
-        The options for a combobox to popup a grid, tree, or dialog were introduced in ARIA 1.1.
-        Changes made in the ARIA 1.1 specification also add support for a code pattern that enables assistive technologies to present the textbox and popup as separately perceivable elements.
-        both ARIA 1.0 and 1.1 patterns are described in the following sections.
-        While using the ARIA 1.1 pattern is recommended as soon as assistive technology support is sufficient,
-        there are no plans to deprecate the ARIA 1.0 pattern.
+        Two other widgets that are also visually compact and enable users to make a single choice from a set of discrete choices are <a href="#Listbox">listbox</a> and <a href="#menubutton">menu button</a>.
+        One feature that distinguishes combobox from both listbox and menu button is that the user's choice can be presented as a value in an editable field, which gives users the ability to select some or all of the value for copying to the clipboard.
+        Comboboxes and menu buttons can be implemented so users can explore the set of allowed choices without losing a previously made choice.
+        That is, users can navigate the set of available choices in a combobox popup or menu and then press <kbd>escape</kbd>, which closes the popup or menu without changing previous input.
```

Comment:
```suggestion
        That is, users can navigate the set of available choices in a combobox popup or menu and then press <kbd>Escape</kbd>, which closes the popup or menu without changing previous input.
```
Capitalize "Escape" for consistency.

Reply:
also, add "the" after "previous" so that it reads, "changing the previous input".